src/Authentication/Authentication/Cmdlets/ConnectGraph.cs

```diff
@@ -170,6 +170,7 @@ private void ThrowParameterError(string parameterName)
         /// </summary>
         public void OnImport()
         {
+            // TODO: Consider checking for a persisted copy of GraphSession or settings on disk.
```

Comment:
should create a workitem or these tend to remain.

Reply:
I'll create a work item for this. This will be useful for people who want to persist their settings such as default profile.

Reply:
#260